pkgs/development/compilers/llvm/git/llvm/default.nix

```diff
@@ -99,6 +99,13 @@ in stdenv.mkDerivation (rec {
     patchShebangs test/BugPoint/compile-custom.ll.py
   '';
 
+  preConfigure = ''
+    # Workaround for configure flags that need to have spaces
+    cmakeFlagsArray+=(
+      -DLLVM_LIT_ARGS='-svj''${NIX_BUILD_CORES} --no-progress-bar'
```

Comment:
Strange, this is a bit surprising to me but IIRC the Nixpkgs abstractions do indeed often strip/replace quotes (among other processing steps or there's a quoting related bug in the code - Bash/shell quoting is unfortunately easy to get wrong...).

It'd be nice if we could avoid that preConfigure workaround but I guess not so that looks fine to me.

Reply:
The Nixpkgs manual [remarks about this detail](https://nixos.org/manual/nixpkgs/stable/#var-stdenv-configureFlagsArray) & [advises to use](https://nixos.org/manual/nixpkgs/stable/#var-stdenv-makeFlagsArray) `*FlagsArray` for these particular cases, `preConfigure` seemed like the best place for this.